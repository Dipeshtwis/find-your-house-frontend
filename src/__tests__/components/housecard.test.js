/* eslint-disable no-unused-vars */
import React from 'react';
import renderer from 'react-test-renderer';
import housecard from '../../components/HouseCard';

it('renders housecard component correctly', () => {
  const housecard = renderer.create(<housecard />).toJSON();
  expect(housecard).toMatchSnapshot();
});
