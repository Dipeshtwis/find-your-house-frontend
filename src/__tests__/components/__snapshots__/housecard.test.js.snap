// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders housecard component correctly 1`] = `<housecard />`;
